dist/build.cmd

```diff
@@ -0,0 +1 @@
+pyinstaller -F -n VivadoLauncher.exe --icon=vivado_logo.ico --clean --distpath=. --workpath=./temp ../pyEDAA/Launcher/__init__.py
```

Comment:
IMHO, generating an executable with the Python entrypoint is out of the scope.
I would prefer if the README and/or the docs were updated first, in order to explain what is the expected purpose and scope of this repository.

Reply:
Can we/ Should we upload a local build executable until the pipe is working for automatic builds?

Reply:
Lately we did updates to the pipeline, so it can run also on macOS, Windows and MinGW64. So we can now start to write the job for it.